plugins/rtmp-services/data/services.json

```diff
@@ -216,6 +216,26 @@
                 "max audio bitrate": 160
             }
         },
+        {
+            "name": "VIMM",
+            "common": false,
```

Comment:
The line can be removed entirely, since the default is false.

Reply:
Will do, thank you.